src/plone/base/interfaces/controlpanel.py

```diff
@@ -1745,6 +1798,44 @@ class IImagingSchema(Interface):
         default=51,
     )
 
+    image_srcsets = schema.JSONField(
+        title=_("Image srcset's"),
+        description=_(
+            "Enter a JSON-formatted image srcset configuration. "
+        ),
+        schema=IMAGE_SRCSET_SCHEMA,
+        default={
+                "large": {
+                    "title": "Large",
+                    "preview": "++theme++barceloneta/static/preview-image-large.png",
```

Comment:
This mentions barceloneta, I am not really sure we want to have it in `plone.base`

Reply:
Well. difficult. I think we can go with it, but it is a bit ugly to forward reference defaults.

Reply:
Thinking further, maybe it would be better to not have any complex defaults in here and configure them in the XML in Products.CMFPlone?

Reply:
I think that it would be better to have this setup by barceloneta/volto generic setup.

Reply:
so far it's mor of an example, we can leave this property out of core anyway.
This is meant to have a preview in the UI.

Reply:
maybe it is time for a site profile package in the core? So for classic we could have this and for volto there can be others? Usually it is done out of core for site installations but here we will have different setups. Worth it?